src/local/centralized_metrics.py

```diff
@@ -0,0 +1,52 @@
+from omegaconf import DictConfig, OmegaConf
+import mlflow
+import hydra
+
+from local.experiment import LassoNetExperiment
+
+
+@hydra.main(config_path='configs', config_name='default')
+def centralized_metrics(cfg: DictConfig):
```

Comment:
not sure I understand what this function does, the function name seems to mismatch the docs. 
if this is for testing a local model on a global test set, then I would give it a more generic name and mention (e.g. in docs) that it can be used for evaluating a local model on a global test set